<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulesItem</class>
 <widget class="QWidget" name="RulesItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDateTimeEdit" name="startDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;start time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="endDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;end time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
